PCLConfig.cmake.in

```diff
@@ -211,6 +211,23 @@ macro(find_rssdk)
   find_package(RSSDK)
 endmacro(find_rssdk)
 
+macro(find_realsense2)
+  if(PCL_ALL_IN_ONE_INSTALLER)
+    set(realsense2_DIR "${PCL_ROOT}/3rdParty/librealsense2/lib/cmake/realsense2" CACHE PATH "The directory containing realsense2Config.cmake")
+  elseif(NOT realsense2_DIR)
+    get_filename_component(realsense2_DIR "@REALSENSE2_INCLUDE_DIRS@" PATH)
+    set(realsense2_DIR "${realsense2_DIR}/lib/cmake/realsense2" CACHE PATH "The directory containing realsense2Config.cmake")
+  endif(PCL_ALL_IN_ONE_INSTALLER)
+
+  find_package(realsense2)
```

Comment:
This should be replaced with `find_package(RSSDK2)`. The `if` block below should be completely removed.